internal/components.go

```diff
@@ -64,7 +83,9 @@ func (o *OpBatcher) Run(service *Service, ctx *ExContext) {
 			"--poll-interval=1s",
 			"--num-confirmations=1",
 			"--private-key=0x2a871d0798f97d79848a013d4936a73bf4cc922c825d33c1cf7073dff6d409c6",
-		)
+			"--log.level="+logLevelToRustLogEnv(ctx.LogLevel),
```

Comment:
@makemake-kbo why do we need it as an env variable and as an arg. We should keep one of them.[